.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 composer.lock
 behat.yml
 *.tar.gz
+/.idea/
```

Comment:
I wouldn't add this to the `.gitignore`. A lot of people use PHPStorm, but there are also hundreds of other editors out there, and we cannot exclude all of them. Git offers a global ignore file for this purpose, a developer can add any of their editors' metadata files and folders to it, and git will ignore them for all this user's repositories.

```
# Define `~/.gitignore` as the global ignore file.
$ git config --global core.excludesfile '~/.gitignore'

# Ignore PHPStorm files in every git repository.
$ echo '/.idea/ > ~/.gitignore
```